### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by jotoeri**
So let's go ahead, bump minversion & appversion. :wink:

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 - **🔒 Data under your control!** Unlike in Google Forms, Typeform, Doodle and others, the survey info and responses are kept private on your instance.
 - **🙋 Get involved!** We have lots of stuff planned like more question types, collaboration on forms, [and much more](https://github.com/nextcloud/forms/milestones)!
     ]]></description>
-    <version>2.2.4</version>
+    <version>2.3.0</version>
```
Comment (skjnldsv) [inline appinfo/info.xml:14]: Missing package.json version bump :)

Comment (jotoeri) [reply appinfo/info.xml:14]: Hm, i didn't plan to release yet... This is just to make the migrations run...
Would you anyways bump the package? Or some.other way? Manual running of migrations? 🤔

Comment (skjnldsv) [reply appinfo/info.xml:14]: I would to keep consistent, :)
But not a big deal indeed ;)

**Review by skjnldsv — APPROVED** [review]

File: appinfo/info.xml (hunk 1 above)
Comment (jotoeri) [reply appinfo/info.xml:14]: I'll add it. 👍 
Just had another thought: How about using an alpha-version? That even has the advantage of beeing able to do such step again, but not having shot the version 2.3 already. 🎉

Comment (skjnldsv) [reply appinfo/info.xml:14]: Sure, I'm fine as well!
I'm using beta/alpha on other places. 

Syntax is `2.3.0-alpha.0`. Npm supports it with `npm version prerelease --preid alpha` (and various preXXXXX options)

**Comment by jotoeri** [comment]
Just pushed it to master now, including the tag... https://github.com/nextcloud/forms/commit/600ce2397afb2f162f7e9da0ae0a2c84bbb8b2ea

**Pull request closed without merging**